eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -105,18 +105,35 @@ jobs:
     # Execute only if there is at least one dependency to be installed
     condition: and(succeeded(), not(eq(variables['vcpkg.deps'], '')))
 
-  - task: CMake@1
-    inputs:
-      cmakeArgs: --version
-    displayName: cmake version
-  - task: CMake@1
-    inputs:
-      cmakeArgs: $(build.args) ..
+  # Disable CMake tasks until regression is fixed:
+  # - task: CMake@1
+  #   inputs:
+  #     cmakeArgs: --version
+  #   displayName: cmake version
+  # - task: CMake@1
+  #   inputs:
+  #     cmakeArgs: $(build.args) ..
+  #   displayName: cmake generate
+  # - task: CMake@1
+  #   inputs:
+  #     cmakeArgs: --build .
+  #   displayName: cmake build
+
+  - script: mkdir build
+    displayName: create working directory
+
+  - script: cmake --version
```

Comment:
I kept this information logging around when we were seeing problems with builds on MacOS hosts. Logging the CMake version hasn't helped much at this point because the problems we've fixed relating to MacOS hosts have always happened earlier in the sequence.